web/src/components/organisms/Project/settings/hooks.ts

```diff
@@ -8,14 +8,15 @@ import {
   useDeleteProjectMutation,
 } from "@reearth-cms/gql/graphql-client-api";
 import { useT } from "@reearth-cms/i18n";
-import { useWorkspace } from "@reearth-cms/state";
+import { useWorkspace, useProject } from "@reearth-cms/state";
 
 type Params = {
   projectId?: string;
 };
 
 export default ({ projectId }: Params) => {
   const navigate = useNavigate();
+  const [_, setCurrentProject] = useProject();
```

Comment:
```suggestion
  const [, setCurrentProject] = useProject();
```

Reply:
@KaWaite ,  nice, I got used to `underscore` for unused variables, but this did not come to my mind.